code/script.js

```diff
@@ -1,47 +1,176 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const userInput = document.getElementById('user-input')
+const sendBtn = document.getElementById('send-btn')
+const nameForm = document.getElementById('name-form')
+const startButton = document.getElementById('startButton')
+const startPage = document.getElementById('startPage')
+const inputWrapper = document.getElementById('input-wrapper')
+const messageAudio = new Audio ('./message-pop-alert.mp3');
 
-// Global variables, if you need any, declared here
-
-// Functions declared here
+//FUNCTIONS
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+    console.log("Hello!")
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/fox.svg" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/alienMonster.svg" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
+// add a sound to each sending message
+messageAudio.currentTime = 0
+messageAudio.play()
+
+// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+//to update the option for music style Study
+const updateStudy = () => {
+  const selectStudy = document.getElementById('selectStudy')
+  if (selectStudy.value === 'brain') {
+    userReply(`Brain Power`)
+    setTimeout(playMusic, 1000)
+  } else if (selectStudy.value === 'relaxing') {
+    userReply(`Relaxing`)
+    setTimeout(playMusic, 1000)
+  } else if (selectStudy.value === 'focus') {
+    userReply(`Focus`)
+    setTimeout(playMusic, 1000)
+  } else {
+    alert('Not a valid option!')
+  }
+}
+
+//to update the option for music style Party
+const updateParty = () => {
+  const selectParty = document.getElementById('selectParty')
+  if (selectParty.value === 'ninety') {
+    userReply(`90’s party`)
```

Comment:
When using the bot, maybe these should be in order based on the century :)

Reply:
Yeah that's right. I didn't realize until you said it :)